justfile

```diff
@@ -32,6 +32,11 @@ run *args="":
 run-release *args="":
     RUST_LOG={{env("RUST_LOG", "stratus=info")}} cargo run --bin stratus -- --enable-genesis --enable-test-accounts {{args}}
 
+run-substrate-mock:
+    npm init -y
```

Comment:
You can put `express` in the package.json in the e2e directory or create a new `package.json` in the subdirectory instead of initing a new Node repo every time you want to run the mock.